ecosystem/sep-TBD.md

```diff
@@ -0,0 +1,154 @@
+
+Freezing is an irreversible process once complete (by design), so take extra care when freezing your NFT issuing account. Some use cases _may_ require the option to unfreeze an account at a future date and this is possible if done in advance, prior to the freeze. This can be done on Stellar via a [pre-authorized transaction](https://developers.stellar.org/docs/glossary/multisig/#pre-authorized-transaction) submitted containing a [`SetOptions`][set-options] operation that sets the `masterWeight` back to 1.
+
+### Representing NFTs
+This section provides interoperability guidance around _representing_ NFTs. 
+
+Since NFTs are represented by Stellar assets and are thus "first-class citizens" in the ecosystem, they should leverage existing interoperability layers. Setting up a [SEP-1 `stellar.toml`][sep1] file provides immediate interoperability with all services and wallets on the Stellar ecosystem. It is highly recommended that all assets you issue on Stellar, NFTs or not, follow the [SEP-1][sep1] standard to provide a valid `stellar.toml` file. This grants your NFT a degree of legitimacy, because most Stellar ecosystem services and wallets tend to discard TOML-less assets and/or flag them as spam. There is detailed documentation about how to do that on [this page](https://developers.stellar.org/docs/issuing-assets/publishing-asset-info/).
+
+#### Using SEP-1
+Many of the [SEP-1][sep1] are highly relevant to NFTs. You should include as many of them as is appropriate for your use case:
+
+  * the `code` and `issuer` fields are essential to describe the Stellar asset that represents your NFT
+  * the `name` and `desc` fields provide human-readable information about your NFT
+  * the `fixed_number`, `max_number`, and `is_unlimited` fields are mutually exclusive ways to describe the supply of your NFT
+  * the `display_decimals` field helps render the right quantity of your NFT, since a single unit of it is likely to be represented by one stroop (i.e. "0.0000001") but you'd want to see "1"
+  * the `image` field is how the ecosystem will "draw" your NFT
+
+Note that even if your NFT isn't an image, you may still want to provide a way to represent it as one (like a logo or symbol) to make it stand out. It's good practice to provide an optimized version of the art to allow fast-loading from services and wallets on Stellar.
+
+Here is an example stellar.toml file describing an asset representing a unique digital image, mimicking the JSON metadata file we described [earlier](#describing-your-nft):
+
+```toml
+[DOCUMENTATION]
+ORG_URL="<https://ink.litemint.store>"
+
+[[CURRENCIES]]
+issuer="GAKZGD5BFXZ7P7P45WHTM6DODMOEWWJUSCAIPGYIVIPYQSVR6MM6YR7M"
+code="DEMOASSET"
+name="A demonstration of NFT metadata"
+desc="This is a description of the cool NFT used in this demo."
+image="https://cloudflare-ipfs.com/ipfs/QmXoypizjW3WknFiJnKLwHCnL72vedxjQkDDP1mXWo6uco/I/m/RickAstleyNeverGonnaGiveYouUp7InchSingleCover.jpg"
+fixed_number=1
+display_decimals=7
```

Comment:
SEP-1's `anchor_asset_type` field, and related fields, probably already fit the bill:
https://github.com/stellar/stellar-protocol/blob/623a2d3b0221d1a2896d3a731f3f255a1b97dca1/ecosystem/sep-0001.md#L131-L133

It doesn't exactly match because anchoring specifies that the anchor will give you asset referenced in exchange for your token, where-as with NFTs you hold the token and can retrieve the asset at the same time without giving up your token.

It would be ashame to add another field when `anchor_asset_type` already support things like "real estate".

Reply:
But this is just a type rather than a value; we still need a generic way to refer to the asset itself :thinking:

Reply:
What about a `representation=<url>` field pointing to a URL? If the issue is that `url=<url>` is too generic

Reply:
Representation is pretty generic too. Is this URL intended to be the actual resource, like the final file, or will it be the JSON referenced by the ipfshash?

Reply:
Maybe `url` is fine given that we can't easily pin it down to something else.

Reply:
Just to respond: it should be the actual resource